examples/data_pipeline/pipelines.py

```diff
@@ -166,44 +145,36 @@ def collate_fn(data_batch):
 
 
 def generate_dataset(args):
-    train, test = raw.DATASETS[args.dataset]()
+    train, test = DATASETS[args.dataset]()
     return [_data for _data in train], [_data for _data in test]
 
 
+PIPELINES = {
+    'sentencepiece': build_sp_pipeline,
+    'experimental_torchtext': build_experimental_torchtext_pipeline,
+    'legacy_torchtext': build_legacy_torchtext_vocab_pipeline,
+    'experimental_fasttext': build_experimental_fasttext_vector_pipeline,
+    'legacy_fasttext': build_legacy_fasttext_vector_pipeline,
+    'experimental_pytext_script_vocab': build_experimental_pytext_script_pipeline,
+    'legacy_batch_torchtext': build_legacy_batch_torchtext_vocab_pipeline,
+}
+
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Data procesing pipelines')
     parser.add_argument('--pipeline', type=str, default='sentencepiece',
                         help='The name of pipeline')
     parser.add_argument('--dataset', type=str, default='AG_NEWS',
                         help='Dataset for performance benchmark')
-    parser.add_argument('--spm-filename', type=str, default='m_user.model',
+    parser.add_argument('--spm-filename', type=str, default='text_unigram_25000',
```

Comment:
Why this change?

Reply:
This is for ease of use in default mode. If the user does not have access to spm model, user can simply run the code in default mode that will download one of the pre-trained spm model (text_unigram_25000). This change does not impact previous behavior, i.e if the use indeed specify m_user.model  (with name other that the name in pre-trained spm models) it will work with the user model.